gbadisasm.h

```diff
@@ -8,6 +9,25 @@ enum LabelType
     LABEL_JUMP_TABLE,
 };
 
+enum BranchType
+{
+    BRANCH_TYPE_UNKNOWN,
+    BRANCH_TYPE_B,
+    BRANCH_TYPE_BL,
+};
+
+struct Label
+{
+    uint32_t addr;
+    uint8_t type;
+    uint8_t branchType;
+    uint32_t size;
+    bool processed;
+    bool isFunc; // 100% sure it's a function, which cannot be changed to BRANCH_TYPE_B. 
+    char *name;
+};
+
+extern struct Label *gLabels;
```

Comment:
I think it would be better if you create a function in disasm.c that set the branch type to `BRANCH_TYPE_B` rather than moving the declarations. Do you agree with me?

Reply:
that is a better idea